From: NARUSE, Yui <naruse@users.sourceforge.jp>
Date: Sat, 8 Jan 2011 11:47:09 +0000 (+0900)
Subject: Fix: wrong decode function setting [nkf-bug:55125]
X-Git-Url: http://git.sourceforge.jp/view?p=nkf%2Fnkf.git;a=commitdiff_plain;h=d2f185cc2134b370b49b7a52a5e81b9671ecc43e

Fix: wrong decode function setting [nkf-bug:55125]

Q-encoded UTF-8 was broken.
---

diff --git a/nkf.c b/nkf.c
index 6877afe..6b944e1 100644
--- a/nkf.c
+++ b/nkf.c
@@ -3878,7 +3878,7 @@ static const unsigned char *mime_pattern[] = {
 
 /* 該当するコードの優先度を上げるための目印 */
 nkf_char (*mime_priority_func[])(nkf_char c2, nkf_char c1, nkf_char c0) = {
-    e_iconv, s_iconv, 0, 0, 0, 0,
+    e_iconv, s_iconv, 0, 0, 0, 0, 0,
 #if defined(UTF8_INPUT_ENABLE)
     w_iconv, w_iconv,
 #endif
